tests/api2/jail.py

```diff
@@ -100,37 +77,14 @@ def test_07_verify_fetch_job_state():
             break
 
 
-def test_08_get_bsd_installed_release():
-    global JOB_ID
-    payload = {
-        'resource': 'RELEASE'
-    }
-    results = POST('/jail/list_resource/', payload)
-    JOB_ID = results.json()
-
-
-def test_09_verify_installed_release():
-    global freeze
-    freeze = False
-    global freeze_msg
-    stop_time = time.time() + 600
-    while True:
-        get_job = GET(f'/core/get_jobs/?id={JOB_ID}')
-        assert get_job.status_code == 200, get_job.text
-        job_status = get_job.json()[0]
-        if job_status['state'] in ('RUNNING', 'WAITING'):
-            if stop_time <= time.time():
-                freeze = True
-                freeze_msg = f"Failed to fetch {RELEASE}"
-                assert False, get_job.text
-                break
-            time.sleep(5)
-        else:
-            assert '11.2-RELEASE' in get_job.json()[0]['result'], get_job.text
-            break
+def test_07_verify_FreeBSD_release_is_installed():
+    results = POST('/jail/releases_choices', False)
+    assert results.status_code == 200, results.text
+    assert isinstance(results.json(), dict), results.text
+    assert '11.2-RELEASE' in results.json(), results.text
```

Comment:
Should we be using `RELEASE` variable which was set above and depcits the release we have decided to fetch ?

Reply:
Good idea